docs/source/gbd2017_models/risks/correlated_risks/index.rst

```diff
@@ -0,0 +1,1199 @@
+ 
+.. math::
+	i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
+ 
+Where i_0 is the risk-deleted rate, PAF is the population attributable fraction for a single or multiple risk factors, and  i_GBDis the rate from GBD. A combined PAF for multiple risk factors is the product of 1 minus the independent PAFs. 
+ 
+.. math::
+	1 - \text{PAF}_{\text{Multiplicative}} = \left(1 - \text{PAF}_{\text{Risk1}}\right)\cdot\left(1 - \text{PAF}_{\text{Risk2}}\right)
+	
+This can be written in a slightly different way for each individual in the simulation and what we are calling the Multiplicative approach to combined PAFs:
+ 
+.. math::
+	PAF_{multiplicative} = 1 - (1 - \frac{1}{\frac{1}{n}\sum_{i=1}^{n}RR_1^{e1_i}}) \cdot (1 - \frac{1}{\frac{1}{n}\sum_{i=1}^{n}RR_2^{e2_i}})
+	
+Where RR is a relative risk and e1 & e2 are indicators {0,1} for exposure status to each risk factor for each individual. We are proposing a slight variation on this formulation, what we are calling the Joint approach to combined PAFs: 
+ 
+.. math::
+	PAF_{joint} = 1 - \frac{1}{\frac{1}{n}\sum_{i=1}^{n} RR_1^{e1_i} \cdot RR_2^{e2_i}}
+	
+These two approaches give nearly the same value for the attributable fraction when the exposure to each risk is independent. However, when the exposures to the risk factors are correlated, the so-called Joint approach gives a higher value of the attributable fraction than the Multiplicative (current Vivarium) approach. 
+We have conducted some simple tests to determine how this impacts the final rate of events among individuals. In this test, we calculate a risk deleted rate as:
+ 
+.. math::
+	i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
+ 
+And calculate the attributable fraction in the two ways described above, as a multiplicative and a joint PAF that we compared. 
+ 
+.. math::
+	i_{multi,0} = (1-{PAF_{multi}}) \cdot i_{{GBD}}  
+.. math::
+	i_{joint,0} = (1-{PAF_{joint}}) \cdot i_{{GBD}}
+	
+	
+Each individual then has a rate of event defined by 
+ 
+.. math::
+	i_{e_1, e_2} = i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
+ 
+Where i is the rate for an individual, i0 is the risk deleted rate, RR is a relative risk and e is an indicator {0,1} denoting if the individual is exposed to that risk factor.
+
+Our simple tests show that when evaluating these attributable fractions on individuals with a known rate of event, the joint PAFs return rates nearly identical to the known rate of events. In contrast, the multiplicative PAFs return rates that are too high when the risk exposures are correlated (rate here is 0.5).
```

Comment:
> In contrast, the multiplicative PAFs return rates that are too high when the risk exposures are correlated (rate here is 0.5).

Or too low when the risk exposures are negatively correlated?  At least, that's what it looks like in the plot.